config/environments/production.rb

```diff
@@ -72,6 +72,18 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    port:           ENV['MAILGUN_SMTP_PORT'],
+    address:        ENV['MAILGUN_SMTP_SERVER'],
+    user_name:      ENV['MAILGUN_SMTP_LOGIN'],
+    password:       ENV['MAILGUN_SMTP_PASSWORD'],
+    domain:         'sandboxd79896c52d80494db98522d8a07b71e3.mailgun.org',
+    authentication: :plain,
+  }
+
+  config.action_mailer.default_url_options = { host: "https://toastmastertopics.herokuapp.com" }
```

Comment:
Put this in an environment variable (`DEFAULT_URL_HOST`?), so that you can set in on Heroku to whatever your production domain is.